gdb/printing.py

```diff
@@ -261,20 +261,26 @@ def _get_children(self, max_count: int) -> None:
                     (c.GetName(), gdb.Value(c))
                     for c in self._sbvalue.GetNonSyntheticValue()]
             # Prepend errors (as synthetic children) to the list of children.
+            byte_order = self._sbvalue.GetTarget().GetByteOrder()
+            address_byte_size = self._sbvalue.GetTarget().GetAddressByteSize()
             for error_str in self._captured_errors:
+                error_str = 'GALA_ERROR(%s)GALA_ERROR' % error_str
+                data = lldb.SBData.CreateDataFromCString(
+                    byte_order, address_byte_size, error_str)
+                data_type = self._sbvalue.GetTarget().GetBasicType(
+                    lldb.eBasicTypeChar).GetArrayType(len(error_str))
                 self._children.insert(
                     0,
                     (
                         'LLDB ERROR',
                         # In order for the error to appear as a synthetic child,
-                        # we need to make it a valid C++ expression.
+                        # we need to create a value from it.
                         # So we pack it into a raw string literal with a custom
                         # delimiter to avoid needing to deal with escaping.
```

Comment:
The last two lines here are no longer correct, we're no longer using a raw string literal with a custom delimiter.

You can just remove them and leave it at
```
                        # In order for the error to appear as a synthetic child,
                        # we need to create a value from it.
```